components/lists/Signaling.tsx

```diff
@@ -19,76 +20,98 @@ interface SignalingProps {
-        <View style={styles.header}>
-          <Badge name={displayUsers.length.toString()} />
-          <CustomText size="sm" fontWeight="bold">
-            Friends are signaling
-          </CustomText>
-        </View>
-        <FlatList
-          contentContainerStyle={styles.listContent}
-          style={styles.flatList}
-          data={displayUsers as User[]}
-          keyExtractor={(item) => item.id}
-          renderItem={({ item }) => (
-            <View style={styles.userCard}>
-              <UserAvatar imageUrl={item.imageUrl || 0} />
-              <View>
-                <UserInfo
-                  firstName={item.firstName}
-                  lastName={item.lastName}
-                  username={item.username}
-                />
-              </View>
-            </View>
-          )}
+      <View style={styles.header}>
+        <CustomText size="xl" fontWeight="bold" style={styles.headerText}>
+          Friends
+        </CustomText>
+        {/* TODO: this should redirect to the search screen */}
+        <CustomButton
+          variant="primary"
+          textSize="sm"
+          title="FIND"
+          textStyles={{ fontWeight: 600 }}
         />
       </View>
+      {!availableUsers.length && (
+        <CustomText style={styles.noUsers}>
+          None of your friends on Wavv are available today
+        </CustomText>
+      )}
+      <BottomSheetSectionList
+        sections={[
+          {
+            title: "available users",
+            data: availableUsers,
+            renderItem: ({ item }) => (
+              <View style={styles.userCard}>
+                <UserAvatar imageUrl={item.imageUrl || 0} />
+                <View>
+                  <UserAvailability
+                    firstName={item.firstName}
+                    lastName={item.lastName}
+                    time={item.time}
+                    activity={item.activity}
+                  />
+                </View>
+              </View>
+            ),
+          },
+          {
+            title: "Other users",
+            data: otherusers,
+            renderItem: ({ item }) => (
+              <View style={[styles.userCard, styles.availableUserCard]}>
+                <UserAvatar imageUrl={item.imageUrl || 0} />
+                <View>
+                  <UserInfo
+                    firstName={item.firstName}
+                    lastName={item.lastName}
+                    username={item.username}
+                  />
+                </View>
+              </View>
+            ),
+          },
+        ]}
+        keyExtractor={(item) => item.id}
+      />
     </BottomDrawer>
   )
 })
 
 const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    paddingTop: 16,
-    paddingLeft: 16,
-  },
   header: {
     flexDirection: "row",
-    gap: 10,
     alignItems: "center",
     marginBottom: 20,
+    justifyContent: "space-between",
+    paddingHorizontal: 20,
+  },
+  headerText: {
+    fontFamily: "suisse",
```

Comment:
this is the default font in a CustomText component, there is no need to apply it again